src/components/search-bar.jsx

```diff
@@ -20,25 +33,21 @@ const createFuse = (field) =>
       keys: ["name"],
     }
   );
-const fuses = {
-  name: createFuse("name"),
-  role: createFuse("role"),
-  organization: createFuse("organization"),
-  school: createFuse("school"),
-  courseOfStudy: createFuse("courseOfStudy"),
-};
 
-const getSuggestions = (value) => {
+// Create an array of fuse objects for each separate wave.
+const fuses = waves.map((_, waveIndex) => {
+  const fuseObject = {};
```

Comment:
Really minor nitpick but could we use `Object.fromEntries` instead?

Reply:
Done! Completely forgot about that function — guess I’m too used to ES5.